src/main/java/com/artipie/api/artifactory/AddUpdateUserSlice.java

```diff
@@ -108,11 +108,12 @@ public Response response(final String line, final Iterable<Map.Entry<String, Str
             json -> {
                 final Optional<Pair<Users.User, String>> res;
                 if (json.containsKey(pswd) && json.containsKey(email)) {
-                    List<String> list = Collections.emptyList();
+                    List<String> list = new ArrayList<>(1);
                     if (json.containsKey(groups)) {
                         list = json.getJsonArray(groups).getValuesAs(JsonString.class)
                             .stream().map(JsonString::getString).collect(Collectors.toList());
                     }
+                    list.add("readers");
```

Comment:
@olenagerasimova same as with adding `readers` group to repository, this should be done only on new user creation